src/App.css

```diff
@@ -33,12 +33,12 @@ body {
   width: 40%;
   justify-content: space-between;
   font-size: 18px;
-  font-family: 'Inter', sans-serif;
-  color: rgb(107, 107, 216);
+  font-family: 'Montserrat', sans-serif;
 }
 
 .site-title a {
   text-decoration: none;
+  color: rgb(32, 136, 255);
 }
 
 .navbar {
```

Comment:
- [ ] [OPTIONAL BUT STRONGLY RECCOMENDED] It has been marked optional because it was not mentioned previously, Kindly ensure that the header/nav bar sticks to the top.

- Your design
<img width="964" alt="image" src="https://user-images.githubusercontent.com/41163571/233686741-13654925-a9a8-467e-83e3-b196438bbeb5.png">

- Zeplin's design
<img width="330" alt="image" src="https://user-images.githubusercontent.com/41163571/233687078-e88fea16-89c9-4f5a-b245-e23ff6c2bb35.png">